zgw_consumers/models/services.py

```diff
@@ -113,6 +131,24 @@ def clean(self):
                 {"header_key": _("If header_value is set, header_key must also be set")}
             )
 
+    @property
+    def connection_check(self) -> bool:
+        from zgw_consumers.client import build_client
+
+        try:
+            client = build_client(self)
+            return client.get(
+                self.api_connection_check_path or self.api_root
+            ).status_code
+        except RequestException as e:
+            logger.info(
+                "Encountered an error while performing the connection check to service %s",
+                self,
+                exc_info=e,
+            )
+
+        return None
```

Comment:
your return values do not match the type hints. I'm also curious to see how the `None` return value is displayed in the admin.